commands/check.js

```diff
@@ -18,14 +18,7 @@ module.exports = {
             .toLowerCase();
         // get the guild id from the interaction
         
-
-        if (subdomain.length < 2 || subdomain.length > 64) {
-            const sadEmbed = new EmbedBuilder()
-                .setDescription("The subdomain length must be between 2 and 64 characters.")
-                .setColor("#0096ff");
-            return await interaction.reply({ embeds: [sadEmbed] });    
-        }
-            
```

Comment:
This still needs to be added except with the limits of 1 to 100 characters.

Reply:
Should be fine now